### ISSUE TITLE
🐛  QueryParser incorrectly parses parameter with multiple values for capital letters keys

### ISSUE THREAD

**Issue by AliYusuf95** [gofiber/fiber#1263]
**Fiber version**
v2.5.0

**Issue description**
Based on a previuos issue https://github.com/gofiber/fiber/issues/782 `QueryParser` should parse array. But it's not the case if the key of the param contains a capital letter.

The issue is in this line:
https://github.com/gofiber/fiber/blob/c71da35ddbf6e3a15e65d0b341bba83464a19771/ctx.go#L791

I'm not sure why the `inputFieldName` is converted to lower case.

**Code snippet**

```go
// Field names should start with an uppercase letter
type Data struct {
    Capital []string `query:"Capital"`
    Small   []string `query:"small"`
}

app.Get("/", func(c *fiber.Ctx) error {
        d := new(Data)

        if err := c.QueryParser(d); err != nil {
            return err
        }

        log.Println(len(p.Capital)) // 1
        log.Println(len(p.Small)) // 2

        // ...
})

// Run tests with the following curl command

// curl -X GET "http://localhost:3000/?Capital=shoe,hat&small=shoe,hat"
```

**Comment by ReneWerner87** [comment]
Thanks for the report

Good point, think we should change the comparison

If you want you are welcome to create a pill request.

**Comment by AliYusuf95** [comment]
it's seems already fixed in v2.7.0 https://github.com/gofiber/fiber/pull/1238
